wordpress-mobile/AztecEditor-iOS
# MediaAttachment: Download Mechanism Cleanup

### Details:
In this PR we're updating the MediaAttachment download mechanism.

cc @diegoreymendez 
Ref. #682

### To test:
- Verify the unit tests are still happy
- Run a smoke test over the Sample App:
  - Try adding new assets (Video + Image).
  - Verify the overlay didn't break!
